scripts/scil_btensor_metrics.py

```diff
@@ -185,14 +184,16 @@ def main():
 
     affine = extract_affine(args.in_dwis)
 
-    tol = args.tolerance
-
+    # Note. This script does not currently allow using a separate b0_threshold
```

Comment:
I think we should add a `check_b0_threshold` using `args.tolerance` here. You would simply have to check if `args.b0_threshold` exists in`check_b0_threshold`, and if not, `b0_thr = args.tolerance` (if it exists).